addons/l10n_rs/data/account_tax_group_data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="tax_group_vat_20" model="account.tax.group">
+            <field name="name">General VAT rate 20%</field>
```

Comment:
AFAIK, the names should be shorter, i.e "20%"

Reply:
Indeed,  CHKL showed me the taxonomy task for Belgium, so I'll follow the same guidelines :ok_hand: